cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -52,7 +53,8 @@ public Samples getSamples(Cluster cluster,
                             Set<TopicPartition> assignedPartitions,
                             long startTimeMs,
                             long endTimeMs,
-                            SamplingMode mode) throws MetricSamplingException {
+                            SamplingMode mode,
+                            MetricDef metricDef) throws MetricSamplingException {
```

Comment:
Does the exception `MetricSamplingException` ever thrown? Can we remove it and the corresponding import?

Reply:
This method inherits the interface. Other implementations may throw this exception.